starter_code/bin/seeds.js

```diff
@@ -0,0 +1,112 @@
+require("dotenv").config();
+
+const mongoose = require("mongoose");
+mongoose;
+
+//get the movie model to do our database query
+const Movie = require("../models/Movie.js");
+// const movieData = require(movies);
+
+mongoose
+  .connect("mongodb://localhost/starter-code", { useNewUrlParser: true })
```

Comment:
Here you can modify the last part not to have all your folders called `starter-code` in your database

Reply:
and if you do so, you should do it as well in your `app.js` file